app/whats-new/views/content.njk

```diff
@@ -0,0 +1,25 @@
+{% extends "layouts/base.njk" %}
+
+{% set heading = "What's New" %}
+
+{% block pageTitle %}
+  {{ heading }} – {{ super() }}
+{% endblock %}
+
+{% block content %}  
+  {% if contentBody and contentTitle %}
+    <header>
+      <h1 class="govuk-heading-xl">
+        {{ heading }}
+      </h1>
+    </header>
+    
+    <h2 class="govuk-heading-l">
+      {{ contentTitle }}
+    </h2>
+    
+    {{ contentBody | safe }}
+    {% else %}
+    
```

Comment:
Might need to discuss with @SophieDodds what we want to show here when Contentful isn't available.

Reply:
> Code changes look good, I've just got a few comments. I love how easy it is to get this page up and running with Contentful though!

Yesss greed! once Contentful is setup, its super clean!